_codux/ui-kit/buttons/buttons.board.module.scss

```diff
@@ -25,32 +25,3 @@
         width: 100%;
     }
 }
-
-.trashIcon {
-    height: 27px;
-    padding-top: 2px;
```

Comment:
You had paddings here and there. Does it look fine without them?

Reply:
I think they are fine. Look
With paddings:
<img width="417" alt="Screenshot 2024-10-14 at 17 59 18" src="https://github.com/user-attachments/assets/ecc07889-38e0-44c2-8f2a-8a0c78ee7306">
No paddings:
<img width="415" alt="Screenshot 2024-10-14 at 17 59 44" src="https://github.com/user-attachments/assets/e84470c1-5c4a-44d2-b682-deeb1ce837e5">

Reply:
Ok, I've moved Menu icon 4px down using padding. Now is better:
<img width="443" alt="Screenshot 2024-10-14 at 18 20 01" src="https://github.com/user-attachments/assets/2d406eea-2fcf-4a2d-97f0-86590490ead8">